clients/cobol-lsp-vscode-extension/CHANGELOG.md

```diff
@@ -1,6 +1,31 @@
 # Changelog
 All notable changes to the COBOL Language Support extension are documented in this file.
 
+## [0.12.0](https://github.com/eclipse/che-che4z-lsp-for-cobol/compare/0.11.1...0.12.0) (2020-07-07)
+
+#### Added
+
+- Support for local copybooks
+- Dynamic registration for watchers
```

Comment:
Maybe, we may drop this point? It is a part of the local copybooks support

Reply:
done in latest commit